v22.2/joins.md

```diff
@@ -133,13 +133,24 @@ Hash joins are performed on two tables as follows:
 
 ### Lookup joins
 
-The [cost-based optimizer](cost-based-optimizer.html) decides when it would be beneficial to use a lookup join. Lookup joins are used when there is a large imbalance in size between the two tables, as it only reads the smaller table and then looks up matches in the larger table. A lookup join requires that the right-hand (i.e., larger) table be indexed on the equality column. A [partial index](partial-indexes.html) can only be used if it contains the subset of rows being looked up.
+The [cost-based optimizer](cost-based-optimizer.html) decides when it would be beneficial to use a lookup join. Lookup joins are used when there is a large imbalance in size between the two tables, as it only reads the smaller table and then looks up matches in the larger table. A lookup join usually requires that the right-hand (i.e., larger) table be indexed on an equality column. A [partial index](partial-indexes.html) can only be used if it contains the subset of rows being looked up.
 
 Lookup joins are performed on two tables as follows:
 
 1. CockroachDB reads each row in the small table.
 1. CockroachDB then scans (or "looks up") the larger table for matches to the smaller table and outputs the matching rows.
 
+If the following conditions are met, the optimizer can perform a lookup join on a series of equalities followed by one inequality constraint:
+
+1. The left and right sides of the inequality have identical [types](data-types.html).
+1. The inequality is between an index column and input column, or between an index column and a constant.
+1. If the index column is `DESC` and the inequality is of the form `idxCol < inputCol`, the column type must support `Datum.Prev` without any chance of failing other than for the minimum value for that type.
```

Comment:
`Datum.Prev` isn't in our docs, so I couldn't infer the meaning here. Is there a way to phrase this in a user-facing way, or should the meaning already be clear?